lib/morse_translator.rb

```diff
@@ -40,4 +43,21 @@ def initialize
                     " " => " "}
   end
 
+  def translate(message)
+    translated = []
+    message.downcase.chars.each do |character|
+      translated << @dictionary[character]
+    end
+    translated.join
```

Comment:
What you are doing is creating a container, changing each element somehow, and then storing that in the container. In the future, I want you to recognize this as a map. What you have is good though!